pages/en/network/developing.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Developing
+---
+
+Developers are a core part of The Graph. They are responsible for both building and querying subgraphs, unlocking data for the web3 ecosystem.
+
+## Building Subgraphs
+
+Developers that build subgraphs use the [GraphQL Schema Language](https://graphql.org/learn/schema-language/) to define the data they want to make available.
+
+The Graph CLI tool streamlines subgraph creation. When you initialize your subgraph, all the default files will be created & configured. Developers can use the built-in APIs or add other libraries as dependencies as long as they are compatible with [AssemblyScript](https://github.com/AssemblyScript/assemblyscript). Subgraphs should be thoroughly tested before deployment. Developers can use [MatchStick](https://github.com/LimeChain/matchstick) to test the mapping logic in a sandboxed environment. Once your subgraph is built & tested, it can be deployed to The Graph's decentralized network. However, before anyone can begin querying it, your subgraph must be curated & indexed.
+
+### Curation
+
+Curators use their knowledge of web3 to assess and signal new subgraphs. They look for subgraphs with reliable and relevant data.
```

Comment:
I think it's worth clarifying here that curation need not be done by third parties. Often subgraph developers will signal on their own subgraphs, so that indexers pick them up

Reply:
Going point - adding that now